kart/workdir.py

```diff
@@ -380,6 +384,14 @@ def reset(
             self.write_full_datasets_to_workdir(
                 [target_datasets[d] for d in ds_inserts]
             )
+        # Update the working copy with files that have changed:
+        kart_attachments = os.environ.get("KART_ATTACHMENTS")
+        if kart_attachments == "true":
+            if base_tree and target_tree:
+                self.update_file_diffs(base_tree, target_tree)
+        else:
+            print("No environment variable KART_ATTACHMENTS set to true")
```

Comment:
Don't print anything here - the idea here is that 
a) you can develop against master, which is easier than developing on a feature branch for long periods of time
b) we can continue to release the master branch but without anything noticeably changing, until we're ready to effective turn on KART_ATTACHMENTS for everyone.
Printing out that its turned off will be a noticeable change for users that won't be of any benefit to them.

Reply:
Also, this extra output is causing some tests to fail - easier to remove this output than to fix the tests.